monkey/infection_monkey/monkey.py

```diff
@@ -280,10 +276,7 @@ def _build_puppet(self, credentials_store: IPropagationCredentialsRepository) ->
             "MSSQLExploiter", exploit_wrapper.wrap(MSSQLExploiter), PluginType.EXPLOITER
         )
 
-        zerologon_telemetry_messenger = CredentialsInterceptingTelemetryMessenger(
-            self._telemetry_messenger, credentials_store
-        )
-        zerologon_wrapper = ExploiterWrapper(zerologon_telemetry_messenger, agent_repository)
+        zerologon_wrapper = ExploiterWrapper(self._telemetry_messenger, agent_repository)
```

Comment:
This isn't needed anymore. We only had a separate wrapper for Zerologon because it stole credentials. We can remove this now.